src/pymor/models/iosys.py

```diff
@@ -933,6 +935,145 @@ def hankel_norm(self, mu=None):
         """
         return self.hsv(mu=mu)[0]
 
+    @cached
+    def l2_norm(self, ast_pole_data={'k': 50, 'sigma': 0, 'which': 'LM'}, mu=None):
```

Comment:
I'm not sure about using a mutable object (`dict`) as a default value (see [here](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)).